forth_src/debug.fs

```diff
@@ -82,11 +82,14 @@ space 1+ loop drop
 dup $7f and $20 < if drop '.' then
 emit 1+ loop cr loop
 last-dump ! base ! ;
-
 : n last-dump @ dump ;
 
-: more $d6 c@ $18 = if $12 emit
-." more" $92 emit key drop page then ;
+true value more?
```

Comment:
I think I'd leave this enabled always? Do you have any scenario in mind when one would like to set it to false?

Reply:
Just a nicety on top of a nicety. I get that  :)